src/addElement.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] Please avoid using `eslint-disable` to get around ESlint errors. ESlint errors are usually denoting an underlying problem with the way your code is setup up. please remove this ESLint error and address any issues that you may have. I know it's much easier to just disable these errors but it's much better to learn to write clean code.